### ISSUE TITLE
Maybe Add PIA openvpn client for easier use

### ISSUE THREAD

**Issue by johnpyp** [sebgl/htpc-download-box#14]
The vpn configuration section is probably the most difficult part right now, and for people who don't use PIA there isn't much remedy for them.

I assume that the majority of the people following this guide are using PIA if they are using a vpn, otherwise they would need external help from their individual VPN.

It might be valuable to support something like this: https://hub.docker.com/r/qmcgaw/private-internet-access. It is like the openvpn-client we are using right now, except it is made specifically for PIA, making it a lot easier to setup, and allowing it to be fully configured via the .env file.

Adding this as an option seems like a big benefit to PIA users, while not really affecting the amount of work other vpn users would have to do anyways. Thoughts?

**Comment by sebgl** [comment]
It looks like a good project. I've looked at the iptables section which is the one thing I was worried about. It looks pretty good (the approach is slightly different, focusing on IP addresses instead of the tun device).

I agree the current VPN section is quite complicated, but keeping it standard does not restrict it to any particular VPN provider.

I think a first step could be to simply link to this project in the VPN doc section (while keeping the current docker-compose), and indicate how it simplifies the configuration for PIA users.

**Comment by johnpyp** [comment]
Yeah, like I mentioned in the Filebrowser issue, it might be useful to have this in an "optional additions" section and link to it in the vpn section.

**Comment by johnpyp** [comment]
Maybe a stretch, but this might be a good consideration for vpn + downloader combo: https://github.com/haugene/docker-transmission-openvpn

It is based on transmission rather than deluge, though I'm not sure how big of a deal that is. I've heard transmission is the least hungry and the most stable, especially at high volumes of torrents.

The big selling point here is support for pretty much all of the major vpn providers (from what I can tell) with just environment variables which is great. It might be worth testing transmission on its own and considering the switch, I think this would simplify what is probably the least straightforward part of the setup right now.

**Comment by johnpyp** [comment]
Other options, with less vpn flexibility but supporting deluge or qBittorrent. I've heard qBittorrent is better with lots of torrents than deluge is, and personally I've been experience quite a bit of lag with Deluge when removing/interacting with lots of torrents at once.

https://github.com/binhex/arch-qbittorrentvpn
https://github.com/binhex/arch-delugevpn
